package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
```

Comment:
We are using `react-bootstrap` for this project. Having another UI component library might create unwanted issues in the future. So, please use components from [`react-bootstrap`](https://react-bootstrap.github.io/components/alerts/)

Reply:
ohkay sure.